packages/client/src/v2-events/features/events/useEvents/procedures/actions/action.ts

```diff
@@ -67,6 +72,42 @@ function errorToastOnConflict(error: TRPCClientError<AppRouter>) {
   }
 }
 
+function getCleanedDeclarationDiff(
```

Comment:
Would be nice to have a comment for this function, which explains what it does exactly